src/fsharp/absil/il.fsi

```diff
@@ -786,6 +786,10 @@ type internal ILMethodBody =
     }
 
 /// Member Access
+/// Assembly - Indicates that the method is accessible to any class of this assembly. (internal)
+/// FamilyAndAssembly - Indicates that the method is accessible to members of this type and its derived types that are in _this assembly only_. (private protected)
```

Comment:
I suggest adding the per-case xmldoc to the specific union cases, like [here](https://github.com/dotnet/fsharp/pull/12334/files?diff=split&w=1#diff-d3a0443592974a1c12ddcfee96b2477edad6de4292d57083d1ddaf11aefa40c6R146-R154).